build/static/style.css

```diff
@@ -36,9 +36,18 @@ td {
 }
 
 .table-legend {
-  margin-left: 10%;
   max-width: 50%;
 }
 .table-header-label {
   text-align: center;
+}
+#report-explanation {
+  font-size: 0.9em;
```

Comment:
are you sure you want to use ems over px?

Reply:
I think so. I like em for things like this sometimes because it's relative, rather than a hard size.